QModManager/Patcher.cs

```diff
@@ -19,6 +19,8 @@ namespace QModManager
     /// </summary>
     public static class Patcher
     {
+        internal static HarmonyInstance harmony;
+
         internal const string IDRegex = "[^0-9a-z_]";
```

Comment:
turn into Regex class

Reply:
Remember that `RegexOptions.Compiled` is not available to .NET 3.5